funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -126,7 +126,14 @@ def stop_endpoint(name: str = typer.Argument("default", autocompletion=complete_
 def restart_endpoint(name: str = typer.Argument("default", autocompletion=complete_endpoint_name)):
     """Restarts an endpoint"""
     manager.stop_endpoint(name)
-    manager.start_endpoint(name)
+
+    endpoint_dir = os.path.join(manager.funcx_dir, name)
```

Comment:
We shouldn't have to duplicate the logic in the start_endpoint function defined above this. We can just invoke those functions, rather than calling through the manager object.